TestPlatform.sln

```diff
@@ -157,6 +157,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "perf", "perf", "{0D4DF78D-7
 		scripts\perf\perfconfig.csv = scripts\perf\perfconfig.csv
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "V1UnitTestProject", "test\TestAssets\V1UnitTestProject\V1UnitTestProject.csproj", "{1F29087C-30F4-4733-8B8A-9226145E8431}"
```

Comment:
Rename `V1UnitTestProject` to `MSTestV1UnitTestProject`.
Add this project to `TestAssets` Sln [here](https://github.com/Microsoft/vstest/blob/master/test/TestAssets/TestAssets.sln/TestAssets.sln).

Reply:
done